lib/DBICx/Sugar.pm

```diff
@@ -95,23 +96,21 @@ sub schema {
             $schema = $schema_class->connect( @conn_info );
         }
     } else {
-        my $dbic_loader = 'DBIx::Class::Schema::Loader';
-        eval { load $dbic_loader };
-        die "You must provide a schema_class option or install $dbic_loader."
-            if $@;
-        $dbic_loader->naming( $options->{schema_loader_naming} || 'v7' );
+        eval { require DBIx::Class::Schema::Loader; 1; }
```

Comment:
I think the old code is cleaner. Please change this part back. Also, please make sure all lines are <= 80 characters long.